packages/palette/src/elements/Image/Image.tsx

```diff
@@ -39,6 +39,7 @@ export const Image = ({
 
 /** Props for a web-only ResponsiveImage component. */
 export interface WebResponsiveImageProps extends ResponsiveImageProps {
+  alt: string
```

Comment:
Realizing this component isn't actually an `img` tag (obviously should be). So will probably have to roll off some of the alts in force once I'm done populating them. But will also have to re-implement this component as an actual image.